src/parser/metrics/Complexity.ts

```diff
@@ -49,7 +51,17 @@ export class Complexity implements Metric {
 
     async calculate(parseFile: ParseFile, tree: Parser.Tree): Promise<MetricResult> {
         const queryBuilder = new QueryBuilder(parseFile, tree);
-        queryBuilder.setStatements(this.mccStatementsSuperSet);
+        switch (parseFile.language) {
+            case Languages.Java:
+                queryBuilder.setStatements(
+                    this.mccStatementsSuperSet.concat(
+                        new SimpleQueryStatement("(class_body (block)) @initBlock")
+                    )
+                );
+                break;
+            default:
+                queryBuilder.setStatements(this.mccStatementsSuperSet);
+        }
```

Comment:
@ce-bo suggested that we should not use switch-cases in a similar situation until we have more than one case to differentiate (see https://github.com/MaibornWolff/metric-gardener/pull/74#discussion_r1479348223), so we should also refactor this to an if-statement here.